runtime/compiler/control/rossa.cpp

```diff
@@ -1971,6 +1974,10 @@ aboutToBootstrap(J9JavaVM * javaVM, J9JITConfig * jitConfig)
       hwProfiler->turnBufferProcessingOffTemporarily();
       }
 
+   printf ("portable shared cache called\n");
+   Trc_JIT_portableSharedCache_enabled_or_disabled(curThread, javaVM->sharedCacheAPI->sharedCachePortable);
```

Comment:
@pshipton Hi Peter, I have a question about tracing points. When I tried `./java -Xtrace:print=j9jit.88 -XX:+PortableSharedCache -XX:-PortableSharedCache -version`
the message doesn't get printed. Is there anything missing in my change?

I tried the same with `./java -Xtrace:print=j9jit.0` which corresponds to `Trc_JIT_VMInitStages_Event1(curThread);` and it's working fine.

Reply:
Oh i might know what's going on:
I probably need to put it after `UT_MODULE_LOADED(J9_UTINTERFACE_FROM_VM(javaVM));`

Reply:
I don't see anything obviously wrong. Is the tracepoint in the lib/J9TraceFormat.dat file?

Reply:
I assume Trc_JIT_VMInitStages_Event1 gets printed before "portable shared cache called"?

Reply:
I think you're looking at my modified version. I previously had `Trc_JIT_portableSharedCache_enabled_or_disabled` just above `UT_MODULE_LOADED(J9_UTINTERFACE_FROM_VM(javaVM));`. I just moved it down to below `Trc_JIT_VMInitStages_Event1`. Maybe this time it will work.

Reply:
Ah, I see. I think the tracepoint is too early, put it after the call to UT_MODULE_LOADED().

Reply:
Note UT_MODULE_LOADED() registers the jit with the tracepoint engine.